Add general entry NFT (#473)
Codecademy/docs: content/general/concepts/nft/nft.md

---
Title: 'NFT'
Description: 'A non-fungible token (NFT) acts as a digital representation to prove ownership for a digital good backed by blockchain technology.'
Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Blockchain'
  - 'Web3'
  - 'Cryptocurrency'
CatalogContent:
  - 'introduction-to-blockchain'
  - 'paths/computer-science'
---

A non-fungible token (NFT) acts as a digital representation to prove ownership for a digital good backed by blockchain technology.

The term "token" in the phrase "non-fungible token" refers to this digital representation. "Non-fungible" simply means "non-replaceable". It can't be replaced, there is only one of them. It is always unique.

For example, a 1 U.S. Dollar note is fungible because you are able to replace that note with another note. However, each NFT is always unique, and you cannot simply swap for another different NFT, even if they are both a representation of the same good. This means that non-fungible things can be far more valuable than fungible things.

> 

Reviewer: ## Minting an NFT

> 

Reviewer: ```suggestion
## Minting an NFT

```

For example, a 1 U.S. Dollar note is fungible because you are able to replace that note with another note. However, each NFT is always unique, and cannot be simply swapped for another different NFT, even if both are a representation of the same good. This means that non-fungible things can be far more valuable than fungible things.

## Minting an NFT

Creating an NFT is known as “minting”. Minting an NFT is the process of taking a piece of content and putting it on a digital registry. In this process, some critical information such as owner name, date, and time is placed on the blockchain. The most common blockchain for minting NFTs is [Ethereum](https://ethereum.org/en). A person can manage any NFTs that the person mints or owns through a crypto wallet.

A crypto wallet is a digital wallet used to store cryptocurrencies and NFTs. Minting or purchasing an NFT can only be done through a crypto wallet. The ownership history of an NFT is easily traceable, verifiable, and can be seen by anyone using the blockchain.

> As of now, NFTs are most often used to prove ownership over a digital good, but they have many other potential uses yet to be explored.

Reviewer 2: ```suggestion
As of now, NFTs are most often used to prove ownership over a digital good, but they have many other potential uses yet to be explored.

Environmental Impact

The amount of electricity consumed for regulating and verifying blockchain transactions leads to massive carbon emissions. It’s estimated that a single NFT transaction (minting or selling) can emit an average of [40kg of CO2](https://earth.org/nfts-environmental-impact/).
```

---
Title: 'NFT'

> Description: 'Non-fungible token which act like a digital representation to prove a ownership for a digital good backed by blockchain technology.'

Reviewer 3: ```suggestion
Description: 'A Non-fungible token (NFT) acts as a digital representation to prove a ownership for a digital good backed by blockchain technology.'
```

Subjects:
  - 'Computer Science'

>   - 'Web Development'

Reviewer 3: Is "Web Development" the best subject for this?  "Cybersecurity" would seem more apt when dealing with the subject of blockchains etc.

Reviewer: Yeah I think that's fine for now. Because we don't have another subject for web3.

> Tags:

Reviewer 3: Review [the "Tags" document](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) and only include tags from there. If you wish to add tags, please insert them in alphabetical order in the tags.md file and include the updated tags.md file in your PR.

If you're adding tags, "cryptocurrency" might be better than "NFT", as it's broader and might cover more entries.

Reviewer 4: ^^^^ 

@SSwiniarski I agree! We should definitely add "cryptocurrency" to `tags.md`. 👍

Reviewer: Added Cryptocurrency, Blockchain, and Web3 tags to **tags.md**

  - 'NFT'
  - 'Blockchain'
CatalogContent:
  - 'paths/front-end-engineer-career-path'
  - 'paths/computer-science'
---

> **NFT**, Non-fungible token which act like a digital representation to prove a ownership for a digital good backed by blockchain technology.

Reviewer 3: ```suggestion
A non-fungible token (NFT) acts as a digital representation to prove a ownership for a digital good backed by blockchain technology.
```

Reviewer: solid change

> The term "token" in the phrase "non-fungible token" refers to this digital representation, and the NFT is just a reference to the original. "Non-fungible" in other word simply means "non-replacable". It can't be replaced, there is only one of them. It is always unique.

Reviewer 3: ```suggestion
The term "token" in the phrase "non-fungible token" refers to this digital representation. "Non-fungible" simply means "non-replaceable". It can't be replaced, there is only one of them. It is always unique.
```

> For example, a 1 GBP note is fungible because you are able to replace that note with another note. However, each NFT is always unique, and you cannot simply swap for another different NFT, even if they are both a representation of the same good. This concludes that non-fungible things are far more valuable than fungible things.

Reviewer 3: ```suggestion
For example, a 1 GBP note is fungible because you are able to replace that note with another note. However, each NFT is always unique, and you cannot simply swap for another different NFT, even if they are both a representation of the same good. This means that non-fungible things can be far more valuable than fungible things.
```

> Creating an NFT is known as “minting”. Minting an NFT is the process of taking a piece of content and putting it on a digital registry. In this process, you are putting some critical information such as owner name, date and time on the blockchain. The most common blockchain for minting NFTs is Ethereum. A person can manage any NFTs that the person mint or own through a crypto wallet.

Reviewer 3: ```suggestion
Creating an NFT is known as “minting”. Minting an NFT is the process of taking a piece of content and putting it on a digital registry. In this process, you are putting some critical information such as owner name, date and time on the blockchain. The most common blockchain for minting NFTs is Ethereum. A person can manage any NFTs that the person mints or owns through a crypto wallet.
```

> A crypto wallet is like a real wallet but it is only used to store cryptocurrencies and NFTs. Minting or purchasing an NFT can only be done through a crypto wallet. The ownership history of an NFT is easily traceable, verifiable and can be seen by anyone.

Reviewer 3: ```suggestion
A crypto wallet is a digital wallet used to store cryptocurrencies and NFTs. Minting or purchasing an NFT can only be done through a crypto wallet. The ownership history of an NFT is easily traceable, verifiable and can be seen by anyone using the blockchain.
```

> As of now, NFTs are most popularly to prove ownership over a digital good but they have many other uses yet to be explored.

Reviewer 3: ```suggestion
As of now, NFTs are most often used to prove ownership over a digital good, but they have many other potential uses yet to be explored.
```

---
Title: 'NFT'

> Description: 'A Non-fungible token (NFT) acts as a digital representation to prove a ownership for a digital good backed by blockchain technology.'

Reviewer: ```suggestion
Description: 'A non-fungible token (NFT) acts as a digital representation to prove ownership for a digital good backed by blockchain technology.'
```

Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Blockchain'
  - 'Web3'
  - 'Cryptocurrency'
CatalogContent:
  - 'introduction-to-blockchain'
  - 'paths/computer-science'
---

> A non-fungible token (NFT) acts as a digital representation to prove a ownership for a digital good backed by blockchain technology.

Reviewer: ```suggestion
A non-fungible token (NFT) acts as a digital representation to prove ownership for a digital good backed by blockchain technology.
```

The term "token" in the phrase "non-fungible token" refers to this digital representation. "Non-fungible" simply means "non-replaceable". It can't be replaced, there is only one of them. It is always unique.

> For example, a 1 GBP note is fungible because you are able to replace that note with another note. However, each NFT is always unique, and you cannot simply swap for another different NFT, even if they are both a representation of the same good. This means that non-fungible things can be far more valuable than fungible things.

Reviewer: ```suggestion
For example, a 1 U.S. Dollar note is fungible because you are able to replace that note with another note. However, each NFT is always unique, and you cannot simply swap for another different NFT, even if they are both a representation of the same good. This means that non-fungible things can be far more valuable than fungible things.
```

---
Title: 'NFT'
Description: 'A non-fungible token (NFT) acts as a digital representation to prove ownership for a digital good backed by blockchain technology.'
Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Blockchain'
  - 'Web3'
  - 'Cryptocurrency'
CatalogContent:
  - 'introduction-to-blockchain'
  - 'paths/computer-science'
---

A non-fungible token (NFT) acts as a digital representation to prove ownership for a digital good backed by blockchain technology.

The term "token" in the phrase "non-fungible token" refers to this digital representation. "Non-fungible" simply means "non-replaceable". It can't be replaced, is one of a kind, and is always unique.

> For example, a 1 U.S. Dollar note is fungible because you are able to replace that note with another note. However, each NFT is always unique, and cannot be simply swapped for another different NFT, even if both are a representation of the same good. This means that non-fungible things can be far more valuable than fungible things.

Reviewer 4: ```suggestion
For example, a single U.S. Dollar note of $1 is fungible because it is able to be replaced with another $1 note. However, each NFT is always unique and cannot be simply swapped for another different NFT, even if both are a representation of the same general good. This means that non-fungible items can be far more valuable than fungible items.
```

## Minting an NFT

> Creating an NFT is known as “minting”. Minting an NFT is the process of taking a piece of content and putting it on a digital registry. In this process, some critical information such as owner name, date, and time is placed on the blockchain. The most common blockchain for minting NFTs is [Ethereum](https://ethereum.org/en). A person can manage any NFTs that the person mints or owns through a crypto wallet.

Reviewer 4: ```suggestion
Creating an NFT is known as “minting”. Minting an NFT is the process of taking a piece of content and putting it on a digital registry, or the blockchain. During this process, some critical information such as owner name, date, and time is placed on the blockchain. The most common blockchain for minting NFTs is [Ethereum](https://ethereum.org/en). A person can manage any NFTs that the person mints or owns through a crypto wallet.
```

> A crypto wallet is a digital wallet used to store cryptocurrencies and NFTs. Minting or purchasing an NFT can only be done through a crypto wallet. The ownership history of an NFT is easily traceable, verifiable, and can be seen by anyone using the blockchain.

Reviewer 4: ```suggestion
A crypto wallet is a digital wallet used to store cryptocurrencies and NFTs. They are the only place where NFTs can be minted or purchased. The ownership history of an NFT is easily traceable, verifiable, and can be seen by anyone using the blockchain.
```

> As of now, NFTs are most often used to prove ownership over a digital good, but they have many other potential uses yet to be explored.

Reviewer 4: ```suggestion
As of now, NFTs are most often used to prove ownership over a digital good, but they have many other potential uses yet to be explored in an ever-growing market.
```
